scripts/run-component-tests.js

```diff
@@ -143,3 +141,13 @@ function correctTypos(target) {
 function convertPathToPosix(pathName) {
   return pathName.replace(/\\/g, '/');
 }
+
+/** Gets the name of the target that should be run. */
+function getTargetName(packageName) {
+  // Schematics don't have _local and browser targets.
+  if (packageName && packageName.endsWith('schematics')) {
```

Comment:
I think ideally we'd check for `schematics` based on what bazel package label has been determined in `getBazelPackageOfComponentName`. That way we could avoid `endsWith`. 

That would be just for making this code more bullet-proof.. not a big deal though 😄

Reply:
Leaving as is because there aren't going to be a lot of changes to schematics. i.e. there won't be any other targets ending with `schematics` that have browser tests.